Spaceport news stories for Hai (#4947)
endless-sky/endless-sky: data/hai/hai news.txt

# Copyright (c) 2020 by Nomadic Volcano
# Based on earlier text copyright (c) 2017 by Michael Zahniser
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

# Visual suggestion: a bit quirky or weird; expression is content, confident, or stoic
news "food in Hai space"
	location
		government "Hai"
	name
		word
			"Visiting chef"
			"Food connoisseur"
			"Tourist"
			"Tour guide"
	portrait
		portrait/human26
		portrait/human31
		portrait/human57
		portrait/human68
	message

> 		word

Reviewer: Might want to add a message about the pickled acorns mentioned in the Centipede's description.

Author: "Back home, my favorite snack was pickled walnuts. Here, the pickled acorns are amazing!"

			"This may sound odd, but the acorn cookies they sell here are the best cookies I've ever eaten."

> 			"A Hai-Human fusion restaurant near here sells peanut pizza. It sounds gross, but the taste is heavenly."

Reviewer: ```suggestion
			"A Hai-human fusion restaurant near here sells peanut pizza. It sounds gross, but the taste is heavenly."
```

			"They make beer from fermented nuts here. I didn't know you could do that!"
			"I just can't get enough acorn stew. These Hai cooks are amazing."
			"Peanut this, walnut that, hazelnut icecream. What I wouldn't give for some good steak right now..."

# Visual suggestion: glasses, clean-cut attire, possibly scientific accoutrements
news "trading in Hai space"
	location
		government "Hai"
	name
		word
			"Merchant captain"
	portrait
		portrait/human02
		portrait/human04
		portrait/human06
		portrait/human10
		portrait/human15
		portrait/human17
		portrait/human31
		portrait/human58
		portrait/human63
		portrait/human72
		portrait/human73
		portrait/human76
		portrait/human79
		portrait/human98
		portrait/human108
		portrait/human112
		portrait/human125
	message
		word
			"Hai Tech is incredible compared to ours. Every time I add a ship to my fleet, I always make a trip up here to stock up on Hai tech."

> 			"Have you tried out one of those Hai anti-missile systems? They'll waste any missile in Human space."

Reviewer: ```suggestion
			"Have you tried out one of those Hai anti-missile systems? They'll waste any missile in human space."
```

			"Can you believe these Hai? They're light-years ahead of us, right under our noses."
			"Have you visited the Quarg ringworld here in Hai space? It's a humbling sight. There's probably more Quarg in there than there are humans anywhere. Makes you feel pretty insignificant."
			"Did you see the stuff they sell on Icelake? Hai technology is amazing."
			"I met some young Hai pilots. It sounds like they start out as poor as we do."

> 			"I sold my entire stock of food at this port. I guess even Hai get tired of nuts."

Reviewer: The implication here is that it's non-Hai food, so maybe the merchant should explicitly state that.
```suggestion
			"I sold my entire stock of food from human space at this port. I guess even Hai get tired of nuts."
```

			"Some Hai were protesting the work week. They want to work only 32 hours a week instead of 40. I used to work 70 hours a week for the Syndicate!"

# Visual suggestion: glasses, clean-cut attire, possibly scientific accoutrements
news "science in Hai space"
	location
		government "Hai"
	name
		word
			"Scientist"
			"Engineer"
	portrait
		portrait/human13
		portrait/human37
		portrait/human43
		portrait/human64
		portrait/human71
		portrait/human83
	message
		word

> 			"A while back, my Hai research partner was nearing a breakthrough, and then she vanished. I haven't seen her since. She probably joined the Unfettered."

Reviewer: Do we really want to outright state "She probably joined the Unfettered?" This idea seems like one of those things that the news should only hint at, not explicitly state, with spaceport mission being where we really tackle the idea.

Author: She didn't join the unfettered. She was kidnapped by the Devil-Run pirates. The purpose of this and some other new spaceport news is to set up the lore for the Devil-Run kidnappings. I removed the reference to the unfettered.

			"Can you believe these Hai? They're light-years ahead of us, right under our noses."
			"Have you visited the Quarg ringworld here in Hai space? It's a humbling sight. There's probably more Quarg in there than there are humans anywhere. Makes you feel pretty insignificant."
			"It's amazing how effective Hai fertilizer has become. If we had that technology, hunger would be a thing of the past."
			"Hai can terraform planets a hundred times faster than we can. I wonder why they didn't do that at Darkwaste?"
			"Hai have the most amazing holographic technology. One of them put on a backpack holoprojector. When he turned it on, he looked like an elephant. The legs and tail even tracked his movements! I wonder if they can mimic humans too?"
			"The Hai have an amazing grasp of quantum chromodynamics. It's why their shield generators are so effective."

> 			"The alloys Hai use in their ships are way beyond our technology."

Reviewer: This blurb feels pretty short compared to the rest. Maybe add a bit extra?

Author: Now, "The alloys Hai use in their ships are way beyond our technology. They're twice as strong with half the weight. No wonder their ships are so fast."

# Visual suggestion: young, normal-looking, more female than male
# or people with animals
news "cute story in Hai space"
	location
		government "Hai"
	name
		word
			"Tourist"
	portrait
		portrait/human00
		portrait/human01
		portrait/human11
		portrait/human18
		portrait/human35
		portrait/human55
		portrait/human66
		portrait/human69
		portrait/human107
		portrait/human114
		portrait/human123
		portrait/human128
	message
		word
			"My little daughter loved her stuffed bear back in human space, but now she wants to hug every Hai she sees instead."

> 			"These Hai are so fluffy I want to hug one every time I see them!"

Reviewer: This reads like a run-on sentence to me. A period, comma, or exclamation point would do to fix that.
```suggestion
			"These Hai are so fluffy! I want to hug one every time I see them!"
```

> 			"I saw the Hai-Human fusion band Nuclear Martini playing on Luna. Nobody realized the hairy drummer was really a Hai in disguise."

Reviewer: ```suggestion
			"I saw the Hai-human fusion band Nuclear Martini playing on Luna. Nobody realized the hairy drummer was really a Hai in disguise."
```

			"I saw a Hai wearing the cutest bonnet the other day. When I get back to human space, I'm definitely getting a pet squirrel."
			"Do you see that Hai woman carrying her baby? Doesn't it look like a long-tailed teddy-bear? The Hai are so cute."

# Visual suggestion: a bit quirky or weird; expression is content, confident, or stoic
news "concerned merchant in Hai space"
	location
		government "Hai"
	name
		word
			"Merchant captain"
	portrait
		portrait/human02
		portrait/human04
		portrait/human06
		portrait/human10
		portrait/human15
		portrait/human17
		portrait/human31
		portrait/human58
		portrait/human63
		portrait/human72
		portrait/human73
		portrait/human76
		portrait/human79
		portrait/human98
		portrait/human108
		portrait/human112
		portrait/human125
	message
		word
			"Don't go too far north up here. The Hai have their own pirates, and they're much more powerful than anything we're used to in human space."
			"I took a shortcut through Wah Ki with six ships. Just two seconds later, BAM! half my fleet is dust."

> 			"I hear Hai space wasn't peaceful a few centuries ago when some reptiles were raiding the place. I wonder where they went?"

Reviewer: ```suggestion
			"I hear Hai space wasn't peaceful a few centuries ago when some reptilian aliens were raiding the place. I wonder where they went?"
```

			"These Hai are so creepy. What are they hiding? I can't wait to get back to human space."
			"The Hai don't like my ship's armory of laser rifles, but I was once boarded by a pirate crew and I'm never letting that happen again! If some overgrown squirrel is going to take away my rifle, it'll have to be from my cold dead hands!"

> 			"I met some young Hai pilots. It sounds like they start out as poor as we do."

Reviewer: This one is already under a section with the same portraits and name. Might as well remove it.

Author: Done.

			"I hope nobody is smuggling Hai technology out of here. I don't want to think about what would happen if the pirates got their hands on this stuff."
			"The Hai will pay you to rush cargo to human space and back. Don't die trying."

# Visual suggestion: a bit quirky or weird; expression is content, confident, or stoic
news "human citizen in Hai space"
	location
		government "Hai"
	name
		word
			"Human citizen"
	portrait
		portrait/human02
		portrait/human04
		portrait/human06
		portrait/human10
		portrait/human15
		portrait/human17
		portrait/human31
		portrait/human58
		portrait/human63
		portrait/human72
		portrait/human73
		portrait/human76
		portrait/human79
		portrait/human98
		portrait/human108
		portrait/human112
		portrait/human125
	message
		word
			"Don't go too far north up here. The Hai have their own pirates, and they're much more powerful than anything we're used to in human space."

> 			"I hear Hai space wasn't peaceful a few centuries ago when some reptiles were raiding the place. I wonder where they went?"

Reviewer: ```suggestion
			"I hear Hai space wasn't peaceful a few centuries ago when some reptilian aliens were raiding the place. I wonder where they went?"
```

> 			"My Hai friend went on one of those prison cruises a while back, but it got lost in an ion storm near Allhome."

Reviewer: What in the world is an ion storm? (We don't have system hazards yet.)

Author: This is to set up lore for the Philosopher's Paradise story. The excuse for the loss is intentionally unbelievable; the prison cruise was a trick and the vacationers were kidnapped.

Author: A fine point. I did all of this just to set up the lore for the Philosopher's Paradise story. Arachi was concerned that there weren't even rumors about people going missing. Adding such rumors belongs in that PR, not this one.

Author: I removed this news story. I'll keep it in the Philosopher's Paradise PR.

> 			"One of my Hai co-workers went to Sunracer to formalize a business deal. I warned her that part of human space is dangerous. I hope I'll see her again."

Reviewer: I'd delete this one. If any Hai are going to human space, it's not to make business deals.

Author: Why not? There's plenty of merchants to transfer cargo and a lot of opportunity for profit. It is risky to go in person to make the deal, but it helps in negotiations. 

There are already several business-related stories and many story-related jobs; and the Hai are an adventurous species. There's no reason to believe Captain `<last>` is the only one getting those spaceport missions. The Captain probably has only talked to a few thousand Hai in a population that is at least in the ones of billions. Statistically speaking, of Captain `<last>` gets, say, one per year, then there are *at least* hundreds such missions per year.

Author: This is to set up lore for the Philosopher's Paradise story which has the player rescue a Hai that was tricked into going to a human planet for business with a non-existent businessman.

Author: I removed this news story. I'll keep it in the Philosopher's Paradise PR.

> 			"A Hai that lives near me went to sleep one night and woke up in a box on Prime. Isn't that weird?"

Reviewer: What?

Author: Deleted.

The kid from Allhome that ended up in a box wasn't the only one. It is to set up lore for the Philosopher's Paradise. I see how this one looks bizarre and out-of-place though.

			"If you think the Hai are weak-willed fur-balls, try watching one of their fencing matches. I've never seen a human move as fast as those Hai."
			"I always said I'd get out of piracy one day, but I know the navy would lock me up for sure. Here in Hai space, I've made a new start."
			"Being in Hai space has taught me to be a better person."
			"Some of my friends abuse the Hai's kindness. After all they've done for me, I'd never do that."
			"Back home, I had to rob people just to get enough money for food. Here I never go hungry."
			"Don't judge me. You have no idea what I had to go through before I got here."
			"I haven't stabbed anyone in months. This is an easier life."
			"Hai are so forgiving here. I got into a fight with one and instead of pressing charges, he had a long talk with me about my choices in life."
			"I always said I'd rethink my life, and now I have. I'm a farmer, not a killer."
			"I used to hate myself for all the terrible crimes I did in human space. Here, the Hai have given me a fresh start."

> 			"You have to re-invent yourself sometimes. I used to be a killer on a pirate ship, now I'm a canner in Hai factory."

Reviewer: ```suggestion
			"You have to re-invent yourself sometimes. I used to be a killer on a pirate ship, now I'm a canner in a Hai factory."
```

			# Duplicated intentionally so pirates say it too:
			"I hope nobody is smuggling Hai technology out of here. I don't want to think about what would happen if the pirates got their hands on that stuff."

> 			"A Hai that lives near me went to sleep one night and woke up in a box on Prime. Isn't that weird?"

Reviewer: Still what?

Author: Deleted.

The kid from Allhome that ended up in a box wasn't the only one. It is to set up lore for the Philosopher's Paradise. I see how this one looks bizarre and out-of-place though.

			"Hai scientists that live near me chartered a flight to Allhome, but it got lost and ended up all the way on Canyon. Some people have the worst sense of direction."

> 			"These Hai are so fluffy I want to hug one every time I see them!"

Reviewer: ```suggestion
			"These Hai are so fluffy! I want to hug one every time I see them!"
```
